src/molecules/menu-item/index.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import styled from "styled-components";
+import colors from "../../atoms/colors";
+import Copy from "../../atoms/copy";
+
+export interface HeaderProps {
+	/** @name Copy */ linkName?: string;
+	/** @name Link */ onClick?: React.MouseEventHandler<HTMLButtonElement>;
+	/** @ignore */ onMouseDown?: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+const StyledMenuItem = styled(Copy)`
+	color: ${colors.white.toString()};
+	margin-left: 32px;
+	cursor: pointer;
+
+	&:first-of-type {
+		margin: 0;
+	}
+
+	@media screen and (max-width: 720px) {
+		display: none;
+	}
+`;
+
+const Menu: React.StatelessComponent<HeaderProps> = (props): JSX.Element => {
```

Comment:
This could be written with a much shorter syntax, but it's ok like this too ;)

```
const Menu: React.StatelessComponent<HeaderProps> = (props): JSX.Element => (
		<StyledMenuItem>
			{props.linkName}
		</StyledMenuItem>
);
```
 or even like this: 

```
const Menu: React.StatelessComponent<HeaderProps> = (props): JSX.Element => <StyledMenuItem>{props.linkName}</StyledMenuItem>;
```